Tools/autotest/autotest.py

```diff
@@ -118,7 +118,7 @@ def build_binaries():
     # copy the script (and various libraries used by the script) as it
     # changes git branch, which can change the script while running
     for thing in ["build_binaries.py", "generate_manifest.py", "gen_stable.py", "build_binaries_history.py",
-                  "board_list.py", "build_sizes.py"]:
+                  "board_list.py", "build_sizes/*"]:
```

Comment:
an install.sh in Tools/scripts/build_sizes that copies to a target dir would be good, we'd copy to topdir/build_sizes

Reply:
Changed to not change autotest.py anymore, as discussed.